src/extensions/yfm/YfmCut/nodeviews/yfm-cut-title.scss

```diff
@@ -1,3 +1,3 @@
-.yfm-cut-title .ye-yfm-cut-title-inner {
+.yfm-cut-title .g-md-yfm-cut-title-inner {
```

Comment:
question (not blocks):
.yfm-cut-title  is is correct?

Reply:
yes, this css selector is correct